lib/CL/pocl_llvm_build.cc

```diff
@@ -353,14 +355,45 @@ int pocl_llvm_build_program(cl_program program,
 
   POCL_MSG_PRINT_LLVM("all build options: %s\n", ss.str().c_str());
 
-  std::istream_iterator<std::string> begin(ss);
-  std::istream_iterator<std::string> end;
-  std::istream_iterator<std::string> i = begin;
-  std::vector<const char*> itemcstrs;
+  char replace_me = 0;
```

Comment:
Also please take note this is a .cc file so we follow the LLVM style (yeah - I'm still sorry for the two different styles), so please make sure the new codes convert these files towards the style guide. It here for example means WSReplacementChar as the variable name that follows the convention.